administrator/language/de-DE/de-DE.com_joomlaupdate.ini

```diff
@@ -74,6 +78,13 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_WARNING_UNKNOWN="Unbekannter Fehler"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_SERVER_ERROR="Update-Server-Fehler"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS="Drittanbieter Erweiterungsprüfungen"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="Es sind keine Erweiterungen installiert."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_BEING_CHECKED="Das System prüft gerade diese Plugins, um herauszufinden, ob während des Upgrades Probleme auftauchen könnten.<br><br>Bitte warten, bis die Prüfungen abgeschlossen sind."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_NON_CORE_PLUGIN_CONFIRMATION="Die Warnungen über möglicherweise inkompatible Plugins ignorieren und mit dem Upgrade fortfahren?"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN="Ein möglicherweise schwerwiegendes Problem."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_CONFIRM_MESSAGE="Die Warnungen über potenziell inkompatible Plugins <strong>wirklich ignorieren</strong> und mit dem Upgrade fortfahren?"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_DESC="Diese Erweiterung enthält ein Plugin, das zu einer fehlerhaften Aktualisierung führen kann.<br><br>Um das Joomla-Upgrade sicher durchzuführen, sollte diese Erweiterung entweder auf eine Version aktualisiert werden, die mit der Zielversion von Joomla kompatibel ist oder das/die betroffene(n) Plugin(s) deaktiviertt und nochmals überprüft werden.<br><br>Weitere Informationen zu den betroffenen Plugins, siehe die Seite „Live-Aktualisierungen“."
```

Comment:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_POTENTIALLY_DANGEROUS_PLUGIN_DESC="Diese Erweiterung enthält ein Plugin, das zu einer fehlerhaften Aktualisierung führen kann.<br><br>Um das Joomla-Upgrade sicher durchzuführen, sollte diese Erweiterung entweder auf eine Version aktualisiert werden, die mit der Zielversion von Joomla kompatibel ist oder das/die betroffene(n) Plugin(s) deaktivieren und erneut überprüfen.<br><br>Weitere Informationen zu den betroffenen Plugins sind im Tab „Live-Update“ zu finden."
```

Reply:
Der Tab heißt Live-Update

Reply:
Ich habe in der 3.9 „Live-Aktualisierungen“ gesehen – wird das in 3.10 geändert?

Reply:
Ja, das haben wir für 3.10 und 4.0 geändert.

Immer direkt in der richtigen Umgebung prüfen.